client/components/reader-full-post-header/index.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import { recordPermalinkClick, recordGaEvent } from 'reader/stats';
+import PostTime from 'reader/post-time';
+import ReaderFullPostHeaderTags from './tags';
+
+const ReaderFullPostHeader = ( { post } ) => {
+	const handlePermalinkClick = ( { } ) => {
+		recordPermalinkClick( 'full_post_title' );
+	};
+
+	const recordDateClick = ( { } ) => {
+		recordPermalinkClick( 'timestamp' );
+		recordGaEvent( 'Clicked Post Permalink', 'timestamp' );
+	};
+
+	return (
+		<div className="reader-full-post-header">
+			{ post.title
+				? <h1 className="reader-full-post-header__title" onClick={ handlePermalinkClick }>
+					<ExternalLink className="reader-full-post-header__title-link" href={ post.URL } target="_blank" icon={ false }>
+						{ post.title }
+					</ExternalLink>
+				</h1>
+				: null }
+			<ul className="reader-full-post-header__meta">
+				{ post.date && post.URL
```

Comment:
I think we want to show the date even if the post has a blank URL? blank urls. how weird.